samples/user-scope-web-application/nodejs/helper/graph-helper.js

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const auth = require("../auth");
+const axios = require('axios');
+var applicationToken = "";
+
+class GraphHelper {
+
+    /**
+     * Get All Group Chats
+     */
+    static async getAllChatsFromGraphpAPI(userId) {
+        let applicationToken = await auth.getAccessToken();
+        let groupChatList = [];
+
+        try {
+            var apiResponse = await axios.get(`https://graph.microsoft.com/v1.0/users/${userId}/chats?notifyOnUserSpecificProperties= ${true}`, {
+                headers: {
+                    "accept": "application/json",
+                    "contentType": 'application/json',
+                    "authorization": "bearer " + applicationToken
+                }
+            });
+
+            groupChatList = [];
+            groupChatList = apiResponse.data.value;
+            if (groupChatList.length > 0) {
+                return groupChatList;
+            }
+        }
+        catch (ex) {
+            return null;
+        }
+    }
+
+    /**
+     * Get All Messages by chatId
+     */
+    static async getAllMessagesByChatId(chatId) {
+        let applicationToken = await auth.getAccessToken();
+        let messagesList = [];
+
+        try {
+            var apiResponse = await axios.get(`https://graph.microsoft.com/v1.0/chats/${chatId}/messages`, {
+                headers: {
+                    "accept": "application/json",
+                    "contentType": 'application/json',
+                    "authorization": "bearer " + applicationToken
+                }
+            });
+
+            messagesList = [];
+            messagesList = apiResponse.data.value;
+
+            if (messagesList.length > 0) {
+                return messagesList;
+            }
+        }
+        catch (ex) {
+            return null;
+        }
+    }
+
+    /**
+     * Delete existing subscription.
+     * @param {string} subscriptionId Id of subscription to be deleted.
+     */
+    static async deleteSubscription(subscriptionId) {
+        await axios.delete(`https://graph.microsoft.com/v1.0/subscriptions/${subscriptionId}`, {
+            headers: {
+                "accept": "application/json",
+                "contentType": 'application/json',
+                "authorization": "bearer " + applicationToken
+            }
+        });
+    }
+
+    /**
+    * id - It can be teams-id (or) chat-id/user-id/teams-app-id
+    * subsId - Is a types of subscription for different resources of graph API.
+    * @param {pageId}. Team and channel subscription.
+    */
+    static async createSubscription(id, token, subsId) {
+        console.log(new Date(Date.now() + 3600000).toISOString())
+        let existing_Subscriptions = null;
+        applicationToken = token;
+        let resource = "";
+        let changeType = "";
+
+        switch (subsId) {
+            case '1':
```

Comment:
instead of numbering, we can use proper text for understanding.

Reply:
yes, updated In all places.